tests/functions.php

```diff
@@ -1,3 +1,168 @@
 <?php
+require_once'../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+    public function testSuccessValidateWeaponName()
+    {
+        $expected = true;
+        $input = ['name' => 'Firestorm', 'damage' => 10];
+        $case = validateWeaponName($input);
+        $this->assertEquals($expected,$case);
+    }
+
```

Comment:
Could add failure test here with $input = [];

Reply:
I don't think this is a failure test.  I'd consider any input array as a proper use of this function